cart.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>sarathi-cart</title>
+    
+    <!-- font awesome cdn link  -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+    
+    <!--  css file link  -->
+    <link rel="stylesheet" href="style.css">
+
+    <!-- My Component Library link -->
+    <link rel="stylesheet" href="https://sarathi-ui.netlify.app/components/component.css">
```

Comment:
If you are using the first and last link in all HTML pages, I will suggest including it in your base.css file.
You can use the below code in base.css:
@import url('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css')
@import url('https://sarathi-ui.netlify.app/components/component.css')

Reply:
Ok I will do that